diff --git a/Makefile b/Makefile
index d7e289e..91224bf 100644
--- a/Makefile
+++ b/Makefile
@@ -13,9 +13,6 @@ ifneq ($(wildcard $(MOOSE_SUBMODULE)/framework/Makefile),)
 else
   MOOSE_DIR        ?= $(shell dirname `pwd`)/moose
 endif
-# Try to use PETSc submodule if PETSC_DIR is not set
-PETSC_DIR          ?=$(MOOSE_DIR)/petsc
-PETSC_ARCH         ?=arch-moose

 # Xolotl
 XOLOTL_DIR         ?= $(CURDIR)/xolotl
@@ -73,20 +70,3 @@ BUILD_EXEC         := yes
 GEN_REVISION       := no
 # DEP_APPS           := $(shell $(FRAMEWORK_DIR)/scripts/find_dep_apps.py $(APPLICATION_NAME))
 include            $(FRAMEWORK_DIR)/app.mk
-
-###############################################################################
-# Additional special case targets should be added here
-
-$(ADDITIONAL_SRC_DEPS): $(XOLOTL_DEPEND_LIBS)
-
-# TODO: should list all source files as a dependency
-# Then if source codes change, make will try to call "cmake"
-# "cmake" should build lib with updated source codes
-$(XOLOTL_DEPEND_LIBS): $(XOLOTL_DIR)/xolotl/solver/src/Solver.cpp
-       cd xolotl; \
-       mkdir build; \
-       cd build; \
-       cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_PREFIX_PATH=$(PETSC_DIR)/$(PETSC_ARCH) \
-       -DBUILD_SHARED_LIBS=yes -DCMAKE_CXX_FLAGS_RELEASE="-O3 -fPIC" -DBUILD_TESTING=OFF \
-       -DCMAKE_INSTALL_PREFIX=$(XOLOTL_DIR)/install ..; \
-       make; make install \
