diff --git a/CMake/BuildPETSc.cmake b/CMake/BuildPETSc.cmake
index ad4f75c5..d8511e51 100644
--- a/CMake/BuildPETSc.cmake
+++ b/CMake/BuildPETSc.cmake
@@ -43,6 +43,8 @@ option(Xolotl_BUILD_LAPACK "Have the PETSc build system build LAPACK" OFF)
 if(NOT Xolotl_BUILD_LAPACK)
     find_package(LAPACK QUIET)
 endif()
+#### HYPRE
+option(Xolotl_BUILD_HYPRE "Have the PETSc build system build HYPRE" OFF)
 
 ## Set options
 set(__script_dir ${CMAKE_SOURCE_DIR}/scripts)
@@ -122,6 +124,11 @@ if(Xolotl_BUILD_LAPACK)
     list(APPEND __build_opts --get-lapack)
     message(STATUS "    - build BLAS/LAPACK")
 endif()
+#### HYPRE
+if(Xolotl_BUILD_HYPRE)
+    list(APPEND __build_opts --get-hypre)
+    message(STATUS "    - build HYPRE")
+endif()
 
 ## Perform build
 set(__output_file "${__external_bin_dir}/petsc_build.out")
